New version: BridgeStan v2.6.2

JuliaRegistrator opened the pull request:
- Registering package: BridgeStan
- Repository: https://github.com/roualdes/bridgestan
- Created by: @WardBrian
- Version: v2.6.2
- Commit: 42e803cff50cb14963a484551d9e859003601214
- Reviewed by: @WardBrian
- Reference: https://github.com/roualdes/bridgestan/commit/42e803cff50cb14963a484551d9e859003601214#commitcomment-155989257
- Description: BridgeStan provides efficient in-memory access through Python, Julia, and R to the methods of a Stan model.
<!-- daf33ad5e6d085e55119bc06703763cec7c0b68dd6e421de144489cbdea59452f02bd83fe8a184914156eaf513823d4d63667f9b89e20044dbd57c20eafe4027809b4d493a57e8a15ec31a5554ef7d450af68980ecaa45c3721546e456aecdd1aace2f275c95509d36ff95f307f61e31e46fa09261cf107fbfc430a229ca0e852745a22b2573f7b63807f518daedc95b927e8f5da85fbcd2bf6698bd2829024050f817e36bab07639a5a97c38b0d443d007fa934c0f1479d6c4bffb7c984c25323514edfbf6fe3ba6f862c036eefe67f379c247ea19a3b3c3d49db23e5100867 -->

WardBrian commented:
[noblock]

This would require some somewhat annoying juggling of our git history to avoid, so I might just let this rot until 2.7.0 is released, hopefully next month...

JuliaRegistrator committed:
New version: BridgeStan v2.6.2

UUID: c88b6f0a-829e-4b0b-94b7-f06ab5908f5a
Repo: https://github.com/roualdes/bridgestan.git
Tree: b17da5b8bc9b073957555ac4d9f8b2170262331f

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
